torch/csrc/jit/passes/frozen_conv_folding.cpp

```diff
@@ -0,0 +1,116 @@
+#include <torch/csrc/jit/passes/frozen_conv_folding.h>
+#include <c10/util/Exception.h>
+#include <torch/csrc/jit/ir/constants.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/dead_code_elimination.h>
+#include <torch/csrc/jit/passes/fold_conv_bn.h>
+#include <torch/torch.h>
+
+namespace torch {
+namespace jit {
+
+using Tensor = at::Tensor;
+
+bool nonConstantParameters(Node* n) {
+  for (size_t i = 1; i < n->inputs().size(); i++) {
```

Comment:
Why do we start from 1 here?

Reply:
It was as of yesterday 😳

Reply:
oop, meant for the `test/elias.py` comment. We start from 1 because the input to conv isn't known, but we do need to know all other values (the conv/batchnorm parameeters)